dali/operators/image/remap/displacement_filter_impl_gpu.cuh

```diff
@@ -208,7 +219,10 @@ class DisplacementFilter<GPUBackend, Displacement,
   explicit DisplacementFilter(const OpSpec &spec) :
       Operator(spec),
       displace_(spec),
-      interp_type_(spec.GetArgument<DALIInterpType>("interp_type")) {
+      interp_type_(spec.GetArgument<DALIInterpType>("interp_type")),
```

Comment:
Order of initialization is wrong - and this could be an assignment inside the constructor body - the type of the variable is trivial whereas the initialization expression is not.

Reply:
Fixed the order.